core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -142,6 +142,52 @@ private static String toSql(RelNode root, SqlDialect dialect) {
     return sqlNode.toSqlString(dialect).getSql();
   }
 
+  @Test public void testSimpleSelectWithOrderByAliasDescNullsFirst() {
```

Comment:
We already have similar tests such as "testHiveSelectQueryWithOrderByAscAndNullsLastShouldBeEmulated". You can add your expectations there